docs/source/deployment/databricks/databricks_notebooks_development_workflow.md

```diff
@@ -0,0 +1,293 @@
+- [Copy project data to DBFS.](#copy-project-data-to-dbfs-using-dbutils)
+- [Modify your project in the Databricks workspace](#modify-your-project-and-test-the-changes)
+
+## Prerequisites
+
+- An active [Databricks deployment](https://docs.databricks.com/getting-started/index.html).
+- A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
+- Python >= 3.7 installed.
+- Git installed.
+- A [GitHub](https://github.com/) account.
+- A Python environment management system installed, [venv](https://docs.python.org/3/library/venv.html), [virtualenv](https://virtualenv.pypa.io/en/latest/) or [Conda](https://docs.conda.io/en/latest/) are popular choices.
+
+## Set up your project
+
+### Install Kedro in a new virtual environment
+
+In your local development environment, create a virtual environment for this tutorial. Any environment management system can be used, though the following commands use Conda:
+
+```bash
+conda create --name iris-databricks python=3.10
+```
+
+Once it is created, activate it:
+
+```bash
+conda activate iris-databricks
+```
+
+With your Conda environment activated, install Kedro:
+
+```bash
+pip install kedro
+```
+
+### Create a new Kedro project
+
+Create a Kedro project with the `databricks-iris` starter using the following command in your local environment:
+
+```bash
+kedro new --starter=databricks-iris
+```
+
+Name your new project `iris-databricks` for consistency with the rest of this guide. This command creates a new Kedro project using the `databricks-iris` starter template.
+
+### Create a GitHub repository
+
+Now you should [create a new repository in GitHub](https://docs.github.com/en/github/getting-started-with-github/create-a-repo) using the official guide. Keep the repository private and don't commit to it yet. For consistency with the rest of this guide, name your GitHub repository `iris-databricks`.
+
+### Create a GitHub personal access token
+
+To synchronise your project between your local development environment and Databricks, you will use a private GitHub repository, which you will create in the next step. For authentication, you will need to create a GitHub personal access token. [Create this token in your GitHub developer settings](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token).
+
+The main steps are:
+
+- Verify your email and navigate to "Settings" under your profile photo.
+- Select "Developer settings" then "Fine-grained tokens" and click on "Generate new token".
+- Select a name and expiration time for your token, choose an expiration time.
+- Select which repositories your token will allow access to and define the token permissions.
+
+```{note}
+Make sure that `repo` scopes are enabled for your token.
+```
```

Comment:
It's my first time using the new beta fine-grained tokens, but I don't see where to enable the "repo" scope. Maybe worth clarifying that?

![image](https://github.com/kedro-org/kedro/assets/316517/76c1da7f-3952-436f-bcd3-31905fa65dff)

Reply:
Or maybe recommend the classic tokens instead. But would be good if we figure it out for the new fine-grained tokens too (just in case the other ones suddenly go away?)

Reply:
You're right, the 'repo scope' note was a holdover from the documentation on the classic tokens and I have removed it :).